feature/profile/src/commonMain/kotlin/com/tunjid/heron/profile/ProfileScreen.kt

```diff
@@ -372,6 +372,7 @@ internal fun ProfileScreen(
                                             if (paneScaffoldState.isSignedOut) signInPopUpState.show()
                                             else actions(Action.UpdatePreferences(update))
                                         },
+                                        onFeedGeneratorMoreOptionsClicked = {},
```

Comment:
Let's remove the `onFeedGeneratorMoreOptionsClicked` lambda from this screen an others. The bottom sheet should be used on the feed and list screens only for now, they're the only screens for sharing records atm.

If a feed is in the timeline, users should got to the feed screen to share it.